libraries/chez/cps/effekt.ss

```diff
@@ -0,0 +1,135 @@
+; Cont a = (a, MetaCont -> #)
+; State = [box]
+; Backup = [(box value)]
+; StackCell = (Prompt State Backup Cont)
+; MetaCont = (Prompt State Backup [StackCell])
```

Comment:
Why do the these have backup fields? Isn't this only needed on captured continuations?